game/src/sandbox/dashboards/commuter.rs

```diff
@@ -320,15 +355,66 @@ impl State for CommuterPatterns {
                 )
                 .named("scale");
                 self.composite.replace(ctx, "scale", new_scale);
-            } else {
-                self.composite.replace(
-                    ctx,
-                    "current",
-                    "None selected".draw_text(ctx).named("current"),
-                );
             }
+        } else {
+            self.composite.replace(
+                ctx,
+                "current",
+                "None selected".draw_text(ctx).named("current"),
+            );
         }
 
+        // Draw outline for Locked Selection
+        match self.selection {
+            BlockSelection::Locked(clicked, _) => {
+                let key = (clicked, BlockDrawableType::BaseLocked);
+                if !self.block_drawables.contains_key(&key) {
+                    let d = ctx.upload(GeomBatch::from(vec![(
+                        Color::BLACK,
+                        self.blocks[clicked]
+                            .shape
+                            .to_outline(Distance::meters(10.0))
+                            .unwrap(),
+                    )]));
+                    self.block_drawables.insert(key, d);
+                }
+            }
+            _ => {}
+        };
+
+        // While selection is locked, draw an overlay with compare_to information for the hovered
+        // block
+        match self.selection {
+            BlockSelection::Locked(clicked, Some(hovered)) => {
+                let key = (hovered, BlockDrawableType::CompareTo);
+                if !self.block_drawables.contains_key(&key) {
+                    let block = &self.blocks[hovered];
+
+                    let border = block.shape.to_outline(Distance::meters(10.0)).unwrap();
+
+                    let mut batch = GeomBatch::from(vec![(Color::WHITE.alpha(0.8), border)]);
+
+                    let base_block = &self.blocks[clicked];
+                    let others = self.count_per_block(&base_block);
+                    let count = others
+                        .into_iter()
+                        .find(|(block, _)| block.id == hovered)
+                        .map(|(_, count)| count)
+                        .unwrap_or(0);
+                    let label_text = format!("{}", count);
```

Comment:
I'd vote for using `abstutil::prettyprint_usize(count)` to help read large numbers. Also in the new `Total: X trips` label.

Reply:
Done!